### DISCUSSION TITLE
Obtaining the predicted value for GP model

### DISCUSSION THREAD

**Discussion by Yoon-SuKyung** [meta-pytorch/botorch#2246] [Q&A]
I want a multi-objective bayesian optimization with two purposes.
New sample points were obtained using qParEGO, qEI, and qEHVI. It was confirmed that the improved pareto front is formed in the figure below. (Yellow dot)
![figure1](https://github.com/pytorch/botorch/assets/98367263/e146c8d0-0692-4f2e-9986-e19fb4776c01)

However, when CFD analysis was performed on the new points(geometry variable combination), it was found that the result was too different from the predicted value. Therefore, i tried to put any x in the surrogate model through the short code below to see if the model was created properly.

![figure2](https://github.com/pytorch/botorch/assets/98367263/d83421c7-bd0b-4317-acf2-469529020e0e)
<train data>

`input_x1=torch.tensor([0.284738096,0.844209842,0.048783983,0.009366032,0.012111966,0.019521414,0.007283044, 0.1])
input_x2=torch.tensor([0.284738096,0.844209842,0.048783983,0.009366032,0.012111966,0.019521414,0.007283044, 0.5])
input_x=torch.stack([input_x1, input_x2])`
`ex_x = normalize(input_x, problem_bounds)`
`ex_obj = model.posterior(ex_x).mean`
`print(ex_obj)`

i got like this.
`[0.4504019021, 0.8916814480],
 [0.4504019021, 0.8916814480]`

However, the output value was strange and very different from the train value. "**model.posterior(ex_x).mean**" was applied to obtain the prediction result of GP by substituting the value of x to the model. In the example, it is calculated through a problem(x) to utilize the equation, but how can the result be obtained if only the surrogate model is created not equation? Or is my GP model not working properly? But when applied to the optimize_qparego_and_get_observation, optimize_ei_and_get_observation, optimize_qehvi_and_get_observation function, I'm confused because I got a result that seems reasonable like a yellow dot. 

In conclusion, how can i obtain the output(predicted value) by substituting the input into the GP model?

[code.txt](https://github.com/pytorch/botorch/files/14568036/code.txt)

**Comment by Balandat** [comment]
What are the two rightmost columns in the table in the txt file? Are these the outcomes from the actual (not the model) evaluation? If so, note that it seems that you have two very different outcomes for the same features, which suggests that there is a lot of noise in the evaluations. In that case predicting an outcome of 0.45 that is between 0.32 and 0.66 is a very reasonable thing to do for a model predicting the mean.
